ciongke/test-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
Would dockerfile style guidelines suggest putting this into a single RUN command?